ydb/library/yql/core/expr_nodes/yql_expr_nodes.json

```diff
@@ -43,7 +43,8 @@
                 {"Index": 1, "Name": "Type", "Type": "TCoAtom"},
                 {"Index": 2, "Name": "Columns", "Type": "TCoAtomList"},
                 {"Index": 3, "Name": "DataColumns", "Type": "TCoAtomList"},
-                {"Index": 4, "Name": "TableSettings", "Type": "TCoNameValueTupleList", "Optional": true}
+                {"Index": 4, "Name": "TableSettings", "Type": "TCoNameValueTupleList"},
```

Comment:
Can you explain please, why this is no longer optional?

Reply:
This very structure can't hold `Optional` in the middle, only in the end. It's a restriction.
But `TableSettings` and `IndexSettings` can be optional independently.
So, I have to remove `Optional` and set both empty values here:
https://github.com/ydb-platform/ydb/pull/6602/files#diff-de6aac4342dbe3e0eae72ee062e1d596762d3d9847dd8f59fafb650edf68dd6eR272